test/FormSelectSpec.tsx

```diff
@@ -0,0 +1,86 @@
+import { render } from '@testing-library/react';
+
+import FormSelect from '../src/FormSelect';
+import FormGroup from '../src/FormGroup';
+
+describe('<FormSelect>', () => {
+  it('should render correctly', () => {
+    const { getByTestId } = render(
+      <FormSelect data-testid="test-id" name="bar" className="my-control" />,
+    );
+
+    const element = getByTestId('test-id');
+    element.tagName.toLowerCase().should.equal('select');
+    element.classList.length.should.equal(2);
+    element.classList.contains('my-control').should.be.true;
+    element.classList.contains('form-select').should.be.true;
+    element.hasAttribute('name').should.be.true;
+    element.getAttribute('name')?.should.equal('bar');
+  });
+
+  it('should render size correctly', () => {
+    const { getByTestId } = render(
+      <FormSelect size="lg" data-testid="test-id" />,
+    );
+
+    const element = getByTestId('test-id');
+    element.classList.length.should.equal(2);
+    element.classList.contains('form-select-lg').should.be.true;
+  });
+
+  it('should render htmlSize correctly', () => {
+    const { getByTestId } = render(
+      <FormSelect htmlSize={3} data-testid="test-id" />,
+    );
+
+    const element = getByTestId('test-id');
+    element.hasAttribute('size');
+    element.getAttribute('size')?.should.equal('3');
```

Comment:
We don't need the `hasAttribute` since we assert the value anyway.  Also better to use non-null assertion operator here since optional chaining may skip the test and provide a false positive.

Reply:
Fixed in the commit https://github.com/react-bootstrap/react-bootstrap/pull/6224/commits/023f9b3e2cffbb507001f992d76bb9cfb4243310